repo readme update (#1049)
brimsec/zq: cmd/pcap/README.md

# `pcap`

> The pcap command indexes and slices pcap files.  Use pcap to create a time index for a large pcap, then derive

Reviewer: ```suggestion
The pcap command indexes and slices pcap files. Use pcap to create a time index for a large pcap, then derive
```

smaller pcaps by efficiently extracting subsets of packets from the large pcap using time range and flow filter

> arguments.  The pcap command was inspired by Vern Paxson's tcpslice program written in the early 1990's.  However,

Reviewer: ```suggestion
arguments. The pcap command was inspired by Vern Paxson's tcpslice program written in the early 1990's. However,
```

> arguments.  The pcap command was inspired by Vern Paxson's tcpslice program written in the early 1990's.  However,

Reviewer 2: ```suggestion
arguments.  The pcap command was inspired by Vern Paxson's [tcpslice](https://github.com/the-tcpdump-group/tcpslice) program written in the early 1990's.  However,
```

> tcpslice does not work with the more sophisticated pcap-ng file format and does not properly handle pcaps with

Reviewer: ```suggestion
tcpslice does not work with the more sophisticated pcapng file format and does not properly handle pcaps with
```
https://wiki.wireshark.org/Development/PcapNg styles it as pcapng.

out-of-order timestamps.

> For all `pcap` options, see the built-in help by running:

Reviewer: ```suggestion
For all `pcap` options, see the built-in help by running
```
Or, if you want a colon, "by running this:" or "by running this command:"
